library-api/src/main/java/org/example/libraryApi/book/controller/BookController.java

```diff
@@ -0,0 +1,97 @@
+package org.example.libraryApi.book.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.example.libraryApi.book.domain.Book;
+import org.example.libraryApi.book.exceptions.BookNotDeletedException;
+import org.example.libraryApi.book.exceptions.BookNotSavedException;
+import org.example.libraryApi.book.exceptions.BookNotTookException;
+import org.example.libraryApi.book.exceptions.BookNotUpdatedException;
+import org.example.libraryApi.book.service.BookCommandService;
+import org.example.libraryApi.book.service.BookQueryService;
+import org.springframework.graphql.data.method.annotation.Argument;
+import org.springframework.graphql.data.method.annotation.MutationMapping;
+import org.springframework.graphql.data.method.annotation.QueryMapping;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+
+@Controller
+@RequiredArgsConstructor
+@Slf4j
+public class BookController {
+
+    private final BookQueryService bookQueryService;
+    private final BookCommandService bookCommandService;
+
+    @QueryMapping
+    public List<Book> getAll(){
```

Comment:
Просмотри, чтобы везде был такой пробел (а вообще, Intellij Idea может автоматически отформатировать код ctrl+alt+L)
```suggestion
    public List<Book> getAll() {
```